### PR TITLE
docs(web): frame capture

### PR DESCRIPTION
**Pull request by arjunattam**

### CODE REVIEW

File: docs/javascript/v2/how--to-guides/record-and-live-stream/frame-capture.mdx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+export const createSnapshotFromVideo = inputVideo => {
+  const canvas = document.createElement("canvas");
+  canvas.width = inputVideo.videoWidth;
+  canvas.height = inputVideo.videoHeight;
+  const context = canvas.getContext("2d");
+  context.drawImage(inputVideo, 0, 0, canvas.width, canvas.height);
+  createSnapshotFromCanvas(canvas);
+};
+```
```
Comment (raviteja83) [inline docs/javascript/v2/how--to-guides/record-and-live-stream/frame-capture.mdx:24]: Let's also add a button with a click handler calling `createSnapshotFromVideo`.

Comment (raviteja83) [reply docs/javascript/v2/how--to-guides/record-and-live-stream/frame-capture.mdx:24]: should we also add react snippet? getting video reference will be slightly different.(using `useRef`)

Comment (arjunattam) [reply docs/javascript/v2/how--to-guides/record-and-live-stream/frame-capture.mdx:24]: yes, please. can you share the snippet here? will make these changes

Comment (raviteja83) [reply docs/javascript/v2/how--to-guides/record-and-live-stream/frame-capture.mdx:24]:
```jsx
const VideoTile = () => {
  const ref = useRef();
 // code to render video
  return ( 
   <video ref={ref} />
   <button onClick={() => {
      createSnapshotFromVideo(ref.current);
   }}>Capture Screenshot</button>
  )
}

```

**Review by raviteja83 — APPROVED** [review]

**Pull request merged**